src/main/java/uk/gov/justice/hmpps/prison/service/AgencyService.java

```diff
@@ -255,7 +255,7 @@ public List<Location> getAgencyEventLocations(final String agencyId, final Strin
 
         final var rawLocations = agencyRepository.getAgencyLocations(agencyId, allEventLocationUsages, orderBy, order);
 
-        return LocationProcessor.processLocations(rawLocations);
+        return LocationProcessor.processLocations(rawLocations).stream().distinct().toList();
```

Comment:
Because the processLocation removes fields there are now duplicates for locations with multiple activity usages.  This is to just make sure the list is distinct